google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -162,14 +162,20 @@ public void checkConfiguration() throws RuntimeConfigurationException {
           "App Engine facet not found in '" + artifact.getName() + "' artifact");
     }
 
+    if (CloudSdkService.getInstance().getSdkHomePath() == null) {
+      throw new RuntimeConfigurationError(
```

Comment:
I'd be curious to know if you can locally run without the facet. I think you can, but the existence of the dev server local run config I think is dependent AE framework support (which adds the AE facet) - but this doesn't mean that removing the facet would cause the local run not to work.